pkg/apiserver/domain/model/model.go

```diff
@@ -118,6 +118,9 @@ func (j *JSONStruct) RawExtension() *runtime.RawExtension {
 		klog.Errorf("yaml to json failure %s", err.Error())
 		return nil
 	}
+	if string(b) == "" || string(b) == "null" {
```

Comment:
Is String(b)== "   " or String(b)=="tab" possible?
If yes, maybe len() == 0 is better.

Reply:
`string(b)` will be `null` and causes some bugs in some cases.